packages/in_app_purchase/in_app_purchase_android/migration_guide.md

```diff
@@ -0,0 +1,113 @@
+<?code-excerpt path-base="excerpts/packages/in_app_purchase_android"?>
```

Comment:
This is missing `_example`; it's the example's package name rather than the plugin's package name.